content/support-matrix/delfin/add-driver-support/_index.md

```diff
@@ -0,0 +1,227 @@
+
+<table>
+    <thead>
+        <tr>
+            <th>Access Type</th>
+            <th>Attributes</th>
+            <th>Additional information</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+           <td rowspan="2"></td>
+           <td></td>
+           <td></td>
+        </tr>
+        <tr>
+            <td></td>
+            <td></td>
+        </tr>
+    </tbody>
+</table>
+
+### Supported alert source registration information
+
+<table>
+    <thead>
+        <tr>
+            <th>Alert Source</th>
+            <th>Supported Protocols</th>
+            <th>Additional information</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+           <td></td>
+           <td></td>
+           <td></td>
+        </tr>
+        <tr>
+            <td></td>
+            <td></td>
+            <td></td>
+        </tr>
+    </tbody>
+</table>
+
+### Performance registration information
+
+<table>
+    <thead>
+        <tr>
+            <th>Resource</th>
+            <th>Default Interval</th>
+            <th>Additional information</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+           <td></td>
+           <td></td>
+           <td></td>
+        </tr>
+        <tr>
+            <td></td>
+            <td></td>
+            <td></td>
+        </tr>
+    </tbody>
+</table>
+
+### Resource data matrix
+
+<table>
+    <thead>
+        <tr>
+            <th>Resource</th>
+            <th>Attribute</th>
+            <th>Additional information</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td rowspan=3></td>
+            <td></td>
+            <td></td>
+        </tr>
+        <tr>
+            <td></td>
+            <td></td>
+        </tr>
+        <tr>
+            <td></td>
+            <td></td>
+        </tr>
+    </tbody>
+</table>
+
+
+### Alert data matrix
+
+<table>
+    <thead>
+        <tr>
+            <th>Attribute</th>
+            <th>Additional information</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+           <td></td>
+           <td></td>
+        </tr>
+        <tr>
+            <td></td>
+            <td></td>
+        </tr>
+    </tbody>
+</table>
+
+### Performance metric data
+
+<table>
+    <thead>
+        <tr>
+            <th>Resource</th>
+            <th>Metric</th>
+            <th>Additional information</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td rowspan=3></td>
+            <td></td>
+            <td></td>
+        </tr>
+        <tr>
+            <td></td>
+            <td></td>
+        </tr>
+        <tr>
+            <td></td>
+            <td></td>
+        </tr>
+        <tr>
+            <td></td>
+            <td></td>
+        </tr>
+    </tbody>
+</table>
+
+```
+10. Raise a PR to the documentation repository(https://github.com/sodafoundation/documentation.git)
+11. Please find sample data filled [template](/support-matrix/delfin/add-driver-support/_index.files/template.odt)
```

Comment:
Give template as a md file.. it should be opened when clicking the link..

Reply:
same as above